js/adapt-contrib-matching.js

```diff
@@ -66,9 +66,12 @@ define([
             this.$('select').select2('destroy');
         },
 
+        /**
+         * Assigns indexes to all the options so that they can safely be randomised
+         */
         setupItemIndexes: function() {
 
-            _.each(this.model.get("_items"), function(item, index) {
+            _.each(this.model.get('_items'), function(item, index) {
```

Comment:
"native" forEach?

Reply:
It certainly could be, I just didn't see any particular advantage to making what would essentially be a breaking change... happy to change if you feel it's better.

Reply:
there are quite a few `_.each` I think on the long run we should migrate all those to native functions.

Reply:
Broadly speaking, I agree - though with the caveat that `Array.forEach` can be a fair bit slower than `for()` (which is what `_.each` uses 'under the hood'). 

Not saying that speed is a factor here, more just that changing to `Array.forEach` wouldn't have any significant advantage over using `_.each` here.

Happy to change to `Array.forEach`, bump version to 3.0 and drop IE8/9/10 from the 'cross platform coverage' list if that's what people prefer...

Reply:
This sounds like an unwinnable conversation.